pkg/kademlia/xor.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package kademlia
+
+import (
+	"fmt"
+	"strconv"
+)
+
```

Comment:
Could you use xorTwoIds (from routing_helpers) to help here?

Reply:
I thought about it but I needed to be concerned with the possibility of mismatched lengths and append is less efficient than direct indexing. I also think it's more readable for something that small but could be convinced otherwise.

Reply:
gotcha!